homeassistant/components/nextdns/switch.py

```diff
@@ -204,6 +204,14 @@ class NextDnsSwitchEntityDescription(
         icon="mdi:cart-outline",
         state=lambda data: data.block_amazon,
     ),
+    NextDnsSwitchEntityDescription[Settings](
+        key="block_bereal",
+        translation_key="block_bereal",
+        entity_category=EntityCategory.CONFIG,
+        entity_registry_enabled_default=False,
```

Comment:
Why do we disable config entities?

Reply:
The idea was not to overwhelm the user with the amount of entities. There are 50 parental control switches. Do you think they should all be enabled by default?

Reply:
That is a lot... ok 👍